documentation/library_guide/macros.rst

```diff
@@ -47,7 +47,7 @@ Macro                              Macro values and the functionality
 ``ONEDPL_HAS_RANGE_ALGORITHMS``    Parallel range algorithms.
 
                                    * ``202409L`` - see :ref:`available algorithms <range-algorithms-202409L>`.
-                                   * ``202505L`` - see :ref:`available algorithms <range-algorithms-202505L>`.
+                                   * ``202509L`` - see :ref:`available algorithms <range-algorithms-202509L>`.
```

Comment:
Should we have some new link here instead of modify existing line?

Reply:
Fixed